scripts/hack/update_codegen.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+trap cleanup EXIT
+GO_CMD=${1:-go}
```

Comment:
nit: shouldn't this ideally be "${1:-go}", which would also keep this more consistent.